uqcsbot/scripts/welcome.py

```diff
@@ -19,7 +19,7 @@
     "Your friendly committee and admins are Nicholas Lambourne @ndl, Jack Caperon @jam, Bradley Stone @Σ, " \
     "Joshua Sutton @jsutton, Ella de Lore @Ella, Matthew Lake @mgtlake, Ishraque Zahin @Ishraque, " \
     "Ryan Kurz @lighthou and Raghav Mishra @rm",
-    "For a list of upcoming events check out the #events channel, or type \"!events\" to see what is coming up"
+    "For a list of upcoming events check out the #events channel, or type \"!events\" to see what is coming up",
```

Comment:
remove the `to see what is coming up` (redundant given the start of the sentence imo)